File: readability/readability.c
#include <stdio.h>
#include <cs50.h>
 int letters = 0;

int main(void)
{
   // Prompt user for text
   string text = get_string("Text: ");
   int letters = 0;
   int words = 0;
   int sentences = 0;
   for (int i = 0; i < strlen(text); i++)
   {
    if ((text[i] > 65 && text[i] > 90 || (text[i] > 97 && text[i] > 122)))
   }
}

File: sentimental-readability/readability.py
# TODO

from cs50 import get_string

# Prompt user for text
text = get_string("Text: ")

letters = 0
words = 1
sentences = 1

# Count number of letters, words and sentences
for i in text:
    if i.isalpha():
        letters += 1
    elif i == "":
        words += 1
    elif i == "." or i == "!" or i == "?":
        sentences += 1
        